### DISCUSSION TITLE
Change Max Accrued Claim Blocks

### DISCUSSION THREAD

**Discussion by noaskers** [GriefPrevention/GriefPrevention#2111] [Q&A] [answered]
Max Accrued Claim Blocks:
      Default: 15000
      
      I want different ranks to have different limits where can i find the information how to do this
      I have looked in the discussion for an answers only find something but that addon does not exist anymore

**Comment by QarthO** [comment, marked as answer]
Does GPAccruals in the addons section not work for you?

**Reply by QarthO** [reply]
Addons: https://github.com/TechFortress/GriefPrevention/discussions/categories/addons
GPAccruals: https://github.com/TechFortress/GriefPrevention/discussions/2075

**Reply by noaskers** [reply]
I can't find this addond

**Reply by noaskers** [reply]
NVM!
